gelato-web3-script/src/web3-functions/GearboxClaimAndBribe/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const GEARBOX_TREE_ADDRESS =
+  "0xA7Df60785e556d65292A2c9A077bb3A8fBF048BC";
+
+export const GEARBOX_MERKLE_BASE_URL =
+  "https://raw.githubusercontent.com/Gearbox-protocol/rewards/master/merkle/";
+
+//todo update with correct deployed contract
+export const CLAIM_AND_BRIBE_CONTRACT =
```

Comment:
Would be better if this could be passed in as a config/constructor from gelato or something.

Reply:
for sure, i'm with you on removing constants. although the only pay to pass in that i know of is with the userArgs. which i setup when the job is schedule, so i guess it's kind of like a constructor (if you have really bad vision)